lib/board.rb

```diff
@@ -0,0 +1,26 @@
+class Board
+  attr_reader :board
```

Comment:
Probably don't need attr_reader

Reply:
I agree. And we can always add it in later if needed.